pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -190,12 +194,13 @@ public Map<String, MercuryOrderDto> getPossibleBarcodesForParticipant(String ddp
 
     @Override
     public int create(MercuryOrderDto mercuryOrderDto) {
-        throw new IllegalStateException("This method should not be used");
+        throw new IllegalStateException("Call the version of this method that includes the raw json payload");
     }
 
     @Override
     public int delete(int id) {
-        return 0;
```

Comment:
It made me sad to see this--having a method do absolutely nothing, with no exceptions or anything...